ci_tutorial/products_pipeline/multi_product.rst

```diff
@@ -0,0 +1,131 @@
+
+Let's take all the 4 different "build-order" files (2 products x 2 configurations each), and merge them together:
+
+.. code-block:: bash
+
+    $ conan graph build-order-merge 
+      --file=game_release.json --file=game_debug.json
+      --file=mapviewer_release.json --file=mapviewer_debug.json
+      --reduce --format=json > build_order.json
+
+
+Now we have applied the ``--reduce`` argument to produce a final ``build_order.json`` that is ready for distribution to the build agents and it only contains those specific packages that need to be built:
+
+.. code-block:: json
+
+    {
+        "order_by": "recipe",
+        "reduced": true,
+        "order": [
+            [
+                {
+                    "ref": "engine/1.0#fba6659c9dd04a4bbdc7a375f22143cb",
+                    "packages": [
+                        [
+                            {
+                                "package_id": "de738ff5d09f0359b81da17c58256c619814a765",
+                                "filenames": ["game_release"],
+                                "build_args": "--requires=engine/1.0 --build=engine/1.0",     
+                            },
+                            {
+                                "package_id": "cbeb3ac76e3d890c630dae5c068bc178e538b090",
+                                "filenames": ["game_debug"],
+                                "build_args": "--requires=engine/1.0 --build=engine/1.0",
+                                
+                            }
+                        ]
+                    ]
+                }
+            ],
+            [
+                {
+                    "ref": "game/1.0#1715574045610faa2705017c71d0000e",
+                    "packages": [
+                        [
+                            {
+                                "package_id": "bac7cd2fe1592075ddc715563984bbe000059d4c",
+                                "filenames": ["game_release"],
+                                "build_args": "--requires=game/1.0 --build=game/1.0",
+                            },
+                            {
+                                "package_id": "01fbc27d2c156886244dafd0804eef1fff13440b",
+                                "filenames": ["game_debug"],
+                                "build_args": "--requires=game/1.0 --build=game/1.0",
+                            }
+                        ]
+                    ]
+                }
+            ]
+        ]
+    }
+
+
+This build order summarizes the necessary builds. First it is necessary to build all different binaries for ``engine/1.0``. This recipe contains 2 different binaries, one for Release and the other for Debug. These binaries belong to the same element in the ``packages`` list, which means they do not depend on each other and can be built in parallel. Each binary tracks its own original build-order file with ``"filenames": ["game_release"],`` so it is possible to deduce the necessary profiles to apply to it.
```

Comment:
Update this line if/when https://github.com/conan-io/conan/pull/17102 is merged

Reply:
Done